src/point_one/fusion_engine/messages/measurements.h

```diff
@@ -316,6 +316,91 @@ struct P1_ALIGNAS(4) RawIMUOutput : public MessagePayload {
   int32_t gyro[3] = {INT32_MAX, INT32_MAX, INT32_MAX};
 };
 
+/**
+ * @brief IMU sensor measurement input  with calibration and corrections applied
+ *        (@ref MessageType::IMU_INPUT, version 1.0).
+ * @ingroup measurement_messages
+ *
+ * This message is an input  from the device containing IMU acceleration and
+ * rotation rate measurements. The measurements been corrected for biases and
+ * scale factors, and have been rotated into the vehicle body frame from the
+ * original IMU orientation, including calibrated mounting error estimates.
+ *
+ * See also @ref RawIMUInput.
+ */
+struct P1_ALIGNAS(4) IMUInput : public MessagePayload {
+  static constexpr MessageType MESSAGE_TYPE = MessageType::IMU_INPUT;
+  static constexpr uint8_t MESSAGE_VERSION = 0;
+
+  /** The time of the measurement, in P1 time (beginning at power-on). */
+  Timestamp p1_time;
+
+  /**
+   * Corrected vehicle x/y/z acceleration (in meters/second^2), resolved in the
+   * body frame.
+   */
+  double accel_mps2[3] = {NAN, NAN, NAN};
+
+  /**
+   * Corrected vehicle x/y/z acceleration standard deviation (in
+   * meters/second^2), resolved in the body frame.
+   */
+  double accel_std_mps2[3] = {NAN, NAN, NAN};
+
+  /**
+   * Corrected vehicle x/y/z rate of rotation (in radians/second), resolved in
+   * the body frame.
+   */
+  double gyro_rps[3] = {NAN, NAN, NAN};
+
+  /**
+   * Corrected vehicle x/y/z rate of rotation standard deviation (in
+   * radians/second), resolved in the body frame.
+   */
+  double gyro_std_rps[3] = {NAN, NAN, NAN};
+};
+
+/**
+ * @brief Raw (uncorrected) IMU sensor measurement input (@ref
```

Comment:
All inputs are uncorrected, so there is no "raw" vs "not" for inputs. This class is not needed.

Reply:
So basically only keep IMUInput? And omit all references to RawIMUInput?